src/GitVersionCore/SemanticVersioning/SemanticVersionFormatValues.cs

```diff
@@ -2,7 +2,7 @@
 using GitVersion.Configuration;
 using GitVersion.Extensions;
 
-namespace GitVersion
+namespace GitVersion.SemanticVersioning
```

Comment:
Why is the `SemanticVersioning` namespace added?

Reply:
renamed to GitVersion namespace